workflow/rules/long_read.smk

```diff
@@ -121,7 +121,7 @@ rule canu_correct:
 
 rule clip_adbc_corrected:
     input:
-        primer=config["preprocessing"]["amplicon-primers"],
+        primer=config["preprocessing"]["amplicon-primers"]["ont"],
```

Comment:
What if no primers are used? This will fail with a key error then.

Reply:
ont + amplicon exclusive